### ISSUE TITLE
Integrate With Nerd Fonts

### ISSUE THREAD

**Issue by sawka** [wavetermdev/waveterm#359]
It would be cool to show the list of nerd fonts, and if one is selected we could do a download + install of the font (automatically) and update the terminal font family.

(issue created off of a comment from youtube)

**Comment by apanchoo** [comment]
Indeed, while waiting to implement the ability to select any font installed on the system, why not just replace the three available fonts with their Nerd Font equivalents? I tried it on my end, and it seems to work well. I would like to make a pull request, but I'm not sure if it aligns with the project's philosophy and if it hasn't already been proposed.
